src/components/NavigatorSidebar.vue

```diff
@@ -169,7 +166,9 @@ export default {
           shownOnMobile: sidenavVisibleOnMobile,
           hiddenOnLarge: sidenavHiddenOnLarge,
         }
-        : {}
+        : {
+          enableSidebar: enableNavigator,
```

Comment:
Can we continue calling it `enableNavigator` instead of `enableSidebar`? Just for consistency.
We may actually want to rename `sidebarProps` to `navigatorProps` and `sidebarListeners` to `navigatorListeners`